doc/nrf/device_guides/working_with_nrf/nrf70/gs.rst

```diff
@@ -366,8 +366,154 @@ For example, you can use the following command when building on the command line
 
 To build for the nRF7002 EK and the nRF7001 or nRF7000 ICs, you can use the corresponding shield name in the above command.
 
+nRF7002 EB
+==========
+
+The nRF7002 Expansion Board (EB) can be used to provide Wi-Fi connectivity to compatible development or evaluation boards through the nRF7002 Wi-Fi 6 companion IC.
```

Comment:
@richabp didn't you intend to use the link to the glossary here?

Reply:
Oh yes, thanks for pointing it out!